README.md

```diff
@@ -1 +1,50 @@
-# portifoliopage
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project is a portifolio page given as a task in the first module
+
+
+## Built With
+
+- Html and Css
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- clone the repo first
+### Setup
+- Open the project in visual studio code
+### Install
+- RUN npm install to install dependencies
+
+## Authors
+
+👤 **Author1**
+- GitHub: [@githubhandle](https://github.com/LDouglasOT)
+- Twitter: [@twitterhandle](https://twitter.com/kawempes_finest)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/luzinda-douglas-69bb811b8/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
[OPTIONAL] This is suppose to have the link of the issues page of this particular repository. In this case it should be pointing to `https://github.com/LDouglasOT/portifoliopage/issues`. Please check and fix that.